third-party/tinyfiledialogs/tinyfiledialogs.c

```diff
@@ -4034,7 +4034,7 @@ static int tkinter2Present(void)
 {
     static int lTkinter2Present = -1 ;
         char lPythonCommand[256];
-        char lPythonParams[256] =
+        char lPythonParams[224] =
```

Comment:
why 224 is better than 256?

Reply:
lPythonParams copied to lPythonCommand.
lPythonParams is constant strings that never reach more than 218 characters (line 4080)